app/helpers/application_helper.rb

```diff
@@ -2,11 +2,33 @@
 
 #= Root-level helpers
 module ApplicationHelper
+
+  def logo_path
+    logo_path = "/assets/images/#{Figaro.env.logo_file}"
+    logo_path
+  end
+
   def logo_tag
     logo_path = "/assets/images/#{Figaro.env.logo_file}"
     image_tag logo_path
   end
 
+  def permissions
+    if Features.wiki_ed? && current_user&.permissions == User::Permissions::NONE
+      'true'
+    else
+      'false'
+    end
+  end
+
+  def languageswitcherenabled
```

Comment:
Nonstandard method name.

Why does this need to return strings? And if strings are really necessary, `Features.enable_language_switcher?.to_s`

Reply:
If its not a string, then i m not able to send it as a parameter.